backend/app/Models/Student.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Notifications\Notifiable;
+use Laravel\Sanctum\HasApiTokens;
+
+class Student extends Model
+{
+    use HasFactory, Notifiable, HasApiTokens;
+
+    // mass assignment
+    protected $fillable = [
+      'name',
+      'username',
+      'email',
+      'thumbnail',
+      'password'
+    ];
```

Comment:
for this one instead of typing all columns as fillable we usually just put the opposite which is 
```
protected $guarded = [];
```

Reply:
Noted Sir. I'll update it right now.